tests/components/v2c/test_diagnostics.py

```diff
@@ -0,0 +1,44 @@
+"""Test V2C diagnostics."""
+
+from unittest.mock import AsyncMock
+
+from syrupy import SnapshotAssertion
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from . import init_integration
+
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+from tests.typing import ClientSessionGenerator
+
+# Fields to exclude from snapshot as they change each run
+TO_EXCLUDE = {
+    "id",
+    "device_id",
+    "via_device_id",
+    "last_updated",
+    "last_changed",
+    "last_reported",
+}
```

Comment:
These values aren't in the diagnostics test right?

Reply:
based myself on existing tests and assumed to much 😓 

<removed>